include/rocksdb/cache.h

```diff
@@ -72,6 +72,17 @@ struct LRUCacheOptions {
   // BlockBasedTableOptions::cache_index_and_filter_blocks_with_high_priority.
   double high_pri_pool_ratio = 0.5;
 
+  // Percentage of cache reserved for low priority entries.
+  // If greater than zero, the LRU list will be split into a high-pri list, a
+  // low-pri list and a bottom-pri list. High-pri entries will be inserted to
+  // the tail of high-pri list, while low-pri entries will be first inserted to
+  // the low-pri list (the midpoint) and bottom-pri entries will be first
+  // inserted to the bottom-pri list.
```

Comment:
We might want to adjust/clarify the wording here a bit (and also for `high_pri_pool_ratio` above); for instance, there are corner cases like the one we hit earlier where `high_pri_pool_ratio == 0` but `low_pri_pool_ratio > 0`. The simplest way of putting it might be saying that `high_pri_pool_ratio > 0` results in a dedicated high-priority pool being created, `low_pri_pool_ratio > 0` results in a dedicated low-priority pool, and items will be inserted into the pool with the highest priority that doesn't exceed the item's priority.